src/components/common/FullViewSnapshotButton.tsx

```diff
@@ -191,9 +191,10 @@ function FullViewSnapshotButton({
             sx={{ height: '2.5rem' }}
             disabled={userGroups ? userGroups.length <= 1 : true}
             options={userGroups ? userGroups.map((group) => group.groupName) : []}
-            name="authorization-domain"
+            name="Select Authorization Domain"
```

Comment:
Changing the name here will affect tests, since that generates the IDs by which we grab the component.

Reply:
Ah, thanks! 
I'm now formatting the id with [lodash's "kebab case"](https://www.geeksforgeeks.org/lodash-_-kebabcase-method/), so hopefully that will at least generate a more meaningful id.